### PR TITLE
Add an intro for the model builder #269

### PR DESCRIPTION
**Pull request by m-mohr**
I started with an overview that users get on "startup" when the model builder is empty. Once you start interacting with the canvas, e.g. by dragging something, it disappears.

image

What do you think? Does this help? Is this confusing? What can we improve? Do the explanations for Python/R work?

### CODE REVIEW

**Comment by soxofaan** [comment]
About the current python instructions:
```
result.to_json()
```

If the user just does this directly in a jupyter cell they get something like this:
Screenshot from 2022-09-01 09-24-57

Which is an escaped string rendering of a JSON string, and basically useless for export/import.
So it's probably better to instruct the user to do 
```
print(result.to_json())
```

FYI: I recently also added `cube.print_json()` which is essentially the same, but that feature is not in a released version yet (will be 0.12.0 probably)

**Comment by soxofaan** [comment]
Another idea is to replace the textual explanation with a GIF or video, where you see collections/processes being dragged

**Comment by m-mohr** [comment]
Good point, I'll update the code for Python. (And then I'm thinking that an export_json() method could make sense that stores it in a temp folder and either downloads it in Jupyter or shows the Path in a Python instance.)

I'm not sure about the GIF. The drag&drop is probably the easiest task and you can't show e.g. the Copy/Paste behavior. Also, if you start to customize the style of your Editor, the GIF may not fit into the style. I think I'd prefer the textual description. We could add a video on YouTube and link to it specifically for the Platform Editor though.

File: src/components/VisualEditor.vue [hunk 1]
```diff
@@ -26,6 +26,28 @@
...
+					<ul>
+						<li>Paste the JSON from your clipboard by clicking <button type="button" @click="paste" title="Paste from clipboard"><i class="fas fa-paste"></i></button> or use <kbd>STRG</kbd> + <kbd>V</kbd> (Windows, Linux) or <kbd>⌘</kbd> + <kbd>V</kbd> (MacOS) when the model builder is in focus.</li>
+						<li>Drag and drop a JSON file from your computer</li>
+						<li>Import a JSON file from your computer or another source such as the internet by clicking <button type="button" @click="importProcess" title="Import process from external source"><i class="fas fa-cloud-download-alt"></i></button></li>
+					</ul>
+					<p>
+						You can also import the processes from the Python and R client.
+						You need to export your process to JSON first:
+						<ul>
+							<li>In Python use <a href="https://open-eo.github.io/openeo-python-client/cookbook/tricks.html#process-graph-export" target="_blank"><code>result.to_json()</code></a></li>
```
Comment (m-mohr) [inline src/components/VisualEditor.vue:43]:
```suggestion
							<li>In Python use <a href="https://open-eo.github.io/openeo-python-client/cookbook/tricks.html#process-graph-export" target="_blank"><code>print(result.to_json())</code></a></li>
```

**Comment by m-mohr** [comment]
>     * I don't see that first clipboard button in the button toolbar, so that is confusing in this context

Yes, it's not there. The button is a compromise because CTRL+V only works once the intro is gone. But I could likely also add the button to the toolbar.

>     * what is STRG? you mean CTRL or something alike

Yeah, so in German it's STRG and in other languages it seems to be CTRL or something else. So I assume I should remove these and just say "paste shortcut" or so.

**Comment by dthiex** [comment]
> Yeah, so in German it's STRG and in other languages it seems to be CTRL or something else. So I assume I should remove these and just say "paste shortcut" or change depending on the language?!

I would not change this depending on the language but simply use "CTRL". The instruction is in English so I find it okay to use the English keyboard names.

Otherwise I went through the updated instruction. I think they are clear (I could easily follow them and they work) and a good addition.

> I'm not sure about the GIF. The drag&drop is probably the easiest task and you can't show e.g. the Copy/Paste behavior. Also, if you start to customize the style of your Editor, the GIF may not fit into the style. I think I'd prefer the textual description. We could add a video on YouTube and link to it specifically for the Platform Editor though.

I would also leave a written description for now, as it's less work to keep them up to date.

File: src/components/VisualEditor.vue [hunk 2]
```diff
@@ -26,6 +26,28 @@
 		<div class="editorSplitter">
 			<DiscoveryToolbar v-if="(showDiscoveryToolbar || isFullScreen) && editable" class="discoveryToolbar" :onAddProcess="insertProcess" />
 			<div class="graphBuilder" @drop="onDrop" @dragover="allowDrop">
+				<div v-if="showHelpOverlay" class="model-overlay">
+					<h2>Welcome!</h2>
+					<p>What you are seeing in this area of the {{ $config.appName }} is the visual model builder.</p>
+					<p>You can start building your model by dragging collections, processes etc. from the left area and dropping them here.</p>
+					<p>Alternatively, you can also import existing processes into the model builder:</p>
+					<ul>
+						<li>Paste the JSON from your clipboard by clicking <button type="button" @click="paste" title="Paste from clipboard"><i class="fas fa-paste"></i></button> or use <kbd>STRG</kbd> + <kbd>V</kbd> (Windows, Linux) or <kbd>⌘</kbd> + <kbd>V</kbd> (MacOS) when the model builder is in focus.</li>
```
Comment (dthiex) [inline src/components/VisualEditor.vue:35]: As mentioned in the comment, I'd just got with "CTRL" here.

**Review by dthiex — CHANGES_REQUESTED** [review]

File: src/components/VisualEditor.vue (hunk 2 above)
Comment (m-mohr) [reply src/components/VisualEditor.vue:35]:
```suggestion
						<li>Paste the JSON from your clipboard by clicking <button type="button" @click="paste" title="Paste from clipboard"><i class="fas fa-paste"></i></button> or use <kbd>CTRL</kbd> + <kbd>V</kbd> (Windows, Linux) or <kbd>⌘</kbd> + <kbd>V</kbd> (MacOS) when the model builder is in focus.</li>
```

**Review by dthiex — APPROVED** [review]

**Pull request merged**